tools/wake/main.cpp

```diff
@@ -168,15 +168,16 @@ void hide_internal_jobs(std::vector<std::vector<std::string>> &out) {
   out.push_back({"tags NOT LIKE '%<d>inspect.visibility=hidden<d>%'", "tags IS NULL"});
 }
 
-void inspect_database(const CommandLineOptions &clo, Database &db, const std::string &wake_cwd) {
-  // tagdag is technically a db inspection, but its very different from the
-  // rest, just handle it and exit.
-  if (clo.tagdag) {
-    JAST json = create_tagdag(db, clo.tagdag);
-    std::cout << json << std::endl;
-    return;
+void query_runs(Database &db)
+{
+  const auto runs = db.get_runs();
+  for (const auto run : runs)
+  {
+    std::cout << run.time.as_string() << " " << run.cmdline << std::endl; 
```

Comment:
Decided to just output the string via `std::cout`, but I could have used the `Describe` library instead like we do with query_jobs : https://github.com/sifive/wake/blob/b4689381eb3bde982e8bad7502b1aca272d9c730/tools/wake/describe.cpp#L371 . But to do this, it would require additional refactoring the whole file to accept not only a `JobReflection` but now a `RunReflection` as well (could make a `TableReflection` parent class that describe takes in for polymorphism). Would that type of change be of benefit and needed now?